config/webpack/blocks/getEntry.js

```diff
@@ -17,6 +17,7 @@ function getEntry (entry/* : ?(string | Object | Array<*>) */) {
       return {
         entry: {
           bundle: [
+            'react-hot-loader/patch',
```

Comment:
it may not be as clean to do this here, but we should probably only add this if `NODE_ENV === 'development'`